firefly-exporter/src/main/java/de/adorsys/opba/fireflyexporter/service/FireFlyTransactionExporter.java

```diff
@@ -50,6 +51,11 @@
     private final TransactionCategorizer categorizer;
     private final TransactionExportJobRepository exportJobRepository;
 
+    @Value("${firefly.pages.page:1}")
```

Comment:
Same as above

Reply:
Deleted unnecessary vars, changed fields to null.

Reply:
fixed